lib/matplotlib/axes/_axes.py

```diff
@@ -4550,6 +4550,20 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
 
             Order of scalars is (left, right, bottom, top).
 
+        Returns
+        -------
+        polycollection : `~matplotlib.collections.PolyCollection`
```

Comment:
remove the name?

Reply:
Yea, I just noticed that because they were moved, I didn't catch some of these.